README.md

```diff
@@ -149,7 +153,7 @@ export JAVA_HOME=/usr/lib/jvm/java
 export HADOOP_CONF_DIR=/etc/hadoop/conf
 ```
  
-* Configure zookeeper in ``conf/slider-client.xml``. In case zookeper is listening on ``master:2181`` you need to add there the following section:
+* Configure zookeeper in ``conf/slider-client.xml``. In case zookeper is listening on ``master:2181`` you need to add there the following section: (not necessary for slider 0.81 unless you have HA enabled in the hadoop confs)
```

Comment:
Since we support only slider -0.80.0, not sure if comments about 0.81 are necessary.